frontend/src/common/authentication/auth-walls/BCeIDAuthWall.tsx

```diff
@@ -35,6 +37,8 @@ export const BCeIDAuthWall = ({
 
   const { userRoles, companyId, isNewBCeIDUser } =
     useContext(OnRouteBCContext);
+  const { data: companyInfo } =
+    useCompanyInfoDetailsQuery(getDefaultRequiredVal(0, companyId));
```

Comment:
you mentioned using an existing context for this `isSuspended` company value @krish1092 - I just could not find it.